Pipdata_script.R

```diff
@@ -0,0 +1,51 @@
+# Test of
+library(devtools)
+load_all()
+
+#----- Temporary load of svy data -----
+
+# Set parameters
+path <- "//tsclient/Y/DLW-OUTPUT" # Path to DLW-QS
```

Comment:
Always use `fs` when working with directories and files. do not user Base R. It will be something like this

```r
path <- Sys.getenv("PIP_ROOT_DIR") |>
  fs::path("DLW-OUTPUT")
```

where `PIP_ROOT_DIR` is an global env variable in your .Renviron that points to the REAL path of the folder: "//w1wbgencifs01/pip/"